clients/da-vinci-client/src/test/java/com/linkedin/davinci/store/rocksdb/RocksDBStorageEngineFactoryTest.java

```diff
@@ -20,7 +21,7 @@ public void testRocksDBCreation() {
 
     RocksDBStorageEngineFactory factory = new RocksDBStorageEngineFactory(serverConfig);
 
-    final String testStore = Utils.getUniqueString("test_store");
+    final String testStore = Version.composeKafkaTopic(Utils.getUniqueString("test_store"), 1);
```

Comment:
So we have seen using wrong resource names in the tests! may be a util validator method should check these resource name and fail early.

Reply:
These are unit tests and previously it can work with any database name, and with the new change, it has to be valid topic resource name.